### PR TITLE
add rule entity in dto

### LINKED ISSUE #376: add rule entity id in dto for violation
### Expected behavior
* add rule entity id in dto for violation

### PR DESCRIPTION
**Pull request by mrandi**
fixes #376

### CODE REVIEW

**Comment by mrandi** [comment]
:+1:

**Comment by harti2006** [comment]
please add a test :stuck_out_tongue:

File: fullstop-web/fullstop-violations-rest/src/main/java/org/zalando/stups/fullstop/web/model/Violation.java [hunk 1]
```diff
@@ -39,6 +39,8 @@
 
     private String username = null;
 
+    private Long ruleID = null;
```
Comment (harti2006) [inline fullstop-web/fullstop-violations-rest/src/main/java/org/zalando/stups/fullstop/web/model/Violation.java]: `= null` is obsolete

File: fullstop-web/fullstop-violations-rest/src/main/java/org/zalando/stups/fullstop/web/converter/ViolationEntityToDtoConverter.java [hunk 2]
```diff
@@ -43,6 +43,8 @@ public Violation convert(ViolationEntity source) {
 
         violation.setUsername(source.getUsername());
 
+        violation.setRuleID(source.getRuleEntity().getId());
```
Comment (harti2006) [inline fullstop-web/fullstop-violations-rest/src/main/java/org/zalando/stups/fullstop/web/converter/ViolationEntityToDtoConverter.java]: NPE :boom:

**Comment by harti2006** [comment]
:+1:

**Comment by ChristianLohmann** [comment]
:+1:

**Pull request merged**